projects/create-a-project.md

```diff
@@ -13,4 +43,3 @@ $ paperspace project init
 ```
 
 This creates a `.ps_project/config.json` file in the current directory to cache information about the project, such as the name, or the last job created.
```

Comment:
```suggestion
This creates a `.ps_project/config.yaml` file in the current directory to cache information about the project, such as the name, or the last job created.
```

Reply:
I believe our config files are only `.yaml` now and not `.json` – @ultrabluewolf can you confirm?

Reply:
That's correct, it's yaml.